test/websocket-server.test.js

```diff
@@ -137,6 +137,25 @@ describe('WebSocketServer', function () {
     });
   });
 
+  describe('#address', function () {
+    it('can provide server address', function (done) {
+      const wss = new WebSocket.Server({ port: 0 }, () => {
+        const addr = wss.address();
+        assert.deepEqual(wss._server.address(), addr);
+        wss.close(done);
+      });
+    });
+
+    it(`will thrown exception if server was closed`, function (done) {
+      const wss = new WebSocket.Server({ port: 0, noServer: true }, () => {
```

Comment:
If you add the `port` option it will create a server. We don't want that.

```js
it('throws an error when operating in "noServer" mode', function () {
  const wss = new WebSocket.Server({ noServer: true });

  assert.throws(
    () => wss.address(),
    /^Error: This server is operating in "noServer" mode$/
  );
});
```